docs/audits/deprecated-api-usage.md

```diff
@@ -0,0 +1,17 @@
+# Avoid using deprecated GPT APIs
+
+## Overview
+
+This audit checks for the presence of warning and error messages coming from GPT which relate to deprecated API usage.
+Deprecated APIs are APIs which are no longer maintained and may be removed. As a result, their use is discouraged, as
```

Comment:
Deprecated APIs are `googletag` methods or configuration options that are no longer maintained or documented. These APIs may be partly or wholly removed from GPT ...

Reply:
Updated